.github/workflows/aws_doc_deploy.yml

```diff
@@ -0,0 +1,37 @@
+name: aws-doc-deploy
+
+on:
+  workflow_run:
+    workflows: ["Build-Docs"]
```

Comment:
This is pretty sweet. It uses the name from any other run CI step as the key and if that completes, it fires? Nice.

Reply:
Ideally, we will see when it gets merged